R/tidyverse.R

```diff
@@ -26,6 +26,12 @@
 #' * `use_tidy_support()`: creates support resources document for GitHub repo
 #'    using SUPPORT.md.
 #'
+#' * `use_tidy_coc()`: creates code of conduct same as use_code_of_conduct
+#'    but for the fact that it's placed in a .github subdirectory.
+#'
+#' *  `use_tidy_community`: wrapper function around contributing, issues,
+#'     support, and code of conduct functions.
+#'
```

Comment:
Can you consistently add `()` when referring to function and enclose in backticks?

Reply:
(>ლ)
Yes!
Sorry, Jenny, you've literally told me that exact same thing before…